src/types/IstioConfigList.ts

```diff
@@ -178,14 +178,15 @@ export const toIstioItems = (istioConfigList: IstioConfigList): IstioConfigItem[
     }
 
     entries.forEach(entry => {
+      const fqdn = entry.metadata.name + '.' + entry.metadata.namespace;
```

Comment:
Looks like combining the name + namespace is done in a few different places to form the validation key. Can this be a separate func so that we don't forget to add the `.` later?

To me, `fqdn` is related to dns which isn't what this var is for. Can this be renamed to something like `validationKey` or perhaps `validationKey(name: string, namespace: string)` if it's a separate func?